src/commands/createVirtualMachine/ConfirmPassphraseStep.ts

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep } from "vscode-azureextensionui";
+import { ext } from "../../extensionVariables";
+import { localize } from "../../localize";
+import { nonNullProp } from "../../utils/nonNull";
+import { IVirtualMachineWizardContext } from "./IVirtualMachineWizardContext";
+
+let mismatchErrorThrown: boolean = false;
```

Comment:
Don't think you meant to make this variable global. That means if I click the back button or start a new wizard it'll still be true for the entire session of that VS Code window